trunk/source/Library-SmtLibUtils/src/de/uni_freiburg/informatik/ultimate/lib/smtlibutils/arrays/MultiDimensionalNestedStore.java

```diff
@@ -183,37 +175,21 @@ public boolean equals(final Object obj) {
 	}
 
 	public static MultiDimensionalNestedStore of(final Term term) {
-		MultiDimensionalNestedStore result = convert1mdseq(term);

```

Comment:
This removal is concerning, as it changed the behavior and caused some errors in the [Nightly build](https://jenkins.sopranium.de/blue/organizations/jenkins/Ultimate%2FUltimate%20Nightly/detail/PR-700/1/tests/). It seems like `Remove redundant comparison statement` was responsible for this, therefore I will disable this settings and re-run the formatter.